FindmallocMC.cmake

```diff
@@ -96,8 +93,14 @@ if(mallocMC_ROOT_DIR)
     string(REGEX MATCH "([0-9]+)" mallocMC_VERSION_PATCH
                                 ${mallocMC_VERSION_PATCH_HPP})
 
+    # mallocMC variables ########################################################
+    #
+    set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR ; mallocMC_INCLUDE_DIRS)
```

Comment:
Ok, I did some light testing. Did you experience any problems with the current implementation?
As far as I can tell, identifiers can be separated by whitespace or `;` interchangeably. If there are multiple separators, it is treated as an empty spot (and not used).
When printing/testing the variable, the following statements all give the same (imho correct) result:

##### List items separated by ; and possibly whitespace
`set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR ; mallocMC_INCLUDE_DIRS)`
`set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR;mallocMC_INCLUDE_DIRS)`
`set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR;;;;;mallocMC_INCLUDE_DIRS)`
`set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR ;;;;; mallocMC_INCLUDE_DIRS)`
`set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR ;; ; ;; mallocMC_INCLUDE_DIRS)`
`set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR ; ; ; ; ; mallocMC_INCLUDE_DIRS)`
##### List items separated by whitespace alone (as seen in your reference)
`set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR mallocMC_INCLUDE_DIRS)`
##### String without whitespace but with explicit list separator
`set(mallocMC_REQUIRED_VARS_LIST "mallocMC_ROOT_DIR;mallocMC_INCLUDE_DIRS")`

All the cases above seem fine to me, except that some are more readable than others (which was my initial intention when adding the spaces around `;`)

--------------

#### What does not work:
##### String separated by whitespace
`set(mallocMC_REQUIRED_VARS_LIST "mallocMC_ROOT_DIR mallocMC_INCLUDE_DIRS")`
(this will create 1 list element that has a space inside)